packages/synapse-react-client/src/components/widgets/query-filter/FacetFilterControls.tsx

```diff
@@ -227,9 +272,30 @@ function FacetFilterControls(props: FacetFilterControlsProps) {
     },
     [facetFiltersShown],
   )
+  const combinedRangeFacetsColumnModelType = combineRangeFacetConfig
+    ? columnModels!.find(
+        model => model.name === combineRangeFacetConfig.minFacetColumn,
+      )?.columnType
+    : undefined
 
   return (
     <div className={`FacetFilterControls`}>
+      {combineRangeFacetConfig && (
+        <CombinedRangeFacetFilter
+          facetResults={combinedRangeFacets as FacetColumnResultRange[]}
+          label={combineRangeFacetConfig.label}
+          columnType={combinedRangeFacetsColumnModelType!}
+          collapsed={false}
+          onChange={(values: (string | number | undefined)[]) =>
+            applyCombinedChangesToRangeColumn(
+              lastRequest,
+              combinedRangeFacets as FacetColumnResultRange[],
+              applyChanges,
+              values as string[],
+            )
+          }
+        ></CombinedRangeFacetFilter>
+      )}
```

Comment:
So will this always be first? If that's not desired it should be an easy future change